boards/components/src/crc.rs

```diff
@@ -0,0 +1,65 @@
+//! Component for CRC syscall interface.
+//!
+//! This provides one Component, `CrcComponent`, which implements a
+//! userspace syscall interface to the CRC peripheral.
+//!
+//! Usage
+//! -----
+//! ```rust
+//! let crc = components::crc::CrcComponent::new(board_kernel, &sam4l::crccu::CRCCU)
+//!     .finalize(components::crc_component_helper!(sam4l::crccu::Crccu));
+//! ```
+
+// Author: Philip Levis <pal@cs.stanford.edu>
+// Last modified: 6/20/2018
```

Comment:
Should these last modified comments be removed? git can tell you when a file was last modified, I'm not sure what this adds.

Reply:
The last time we talked about this, there was light indecision, but the rationale is that while git tells you who last modified files, it's not so great at necessarily capturing who owns a certain component (e.g. you would have to trace blame through Brad's recent clippy patches to likely find the correct owner in several instances). The intent of these was closer to the MAINTAINERS file from Linux; their adherence in practice is a bit hit or miss, but that's their purpose, and I don't think git quite satisfies that.

Reply:
Doesn't that apply to the author comment? Does the last modified add anything to that?

Reply:
Mhm, yeah. I think the intent was to capture the last "substantial" update (i.e. how stable is this thing, has something changed recently if something breaks) -- @phil-levis may have more to say their utility; I'm fairly neutral on them

Reply:
If it's a standard tock thing then it's not an issue for this PR